tests/scripts/installandtest.sh

```diff
@@ -18,14 +24,15 @@ if [ -z "${SKIP_INSTALL}" ]; then
 fi
 $HOME/peak/run.sh ${TESTS_REGEX}
 
```

Comment:
@crobby I moved the pod and operator logs out of the test error check so that we always save these logs as artifacts during a test run. Can I remove `SKIP_PODS_OUTPUT` as an variable since it only appears to be used to toggle to the test log?